ci/cloudbuild/builds/shared.sh

```diff
@@ -24,15 +24,14 @@ source module ci/cloudbuild/builds/lib/features.sh
 export CC=gcc
 export CXX=g++
 
-read -r ENABLED_FEATURES < <(features::list_full_cmake)
 mapfile -t cmake_args < <(cmake::common_args)
 
 INSTALL_PREFIX="/var/tmp/google-cloud-cpp"
 cmake "${cmake_args[@]}" \
   -DCMAKE_INSTALL_PREFIX="${INSTALL_PREFIX}" \
   -DCMAKE_INSTALL_MESSAGE=NEVER \
   -DBUILD_SHARED_LIBS=ON \
-  -DGOOGLE_CLOUD_CPP_ENABLE="${ENABLED_FEATURES}"
+  -DGOOGLE_CLOUD_CPP_ENABLE="__ga_libraries__,__experimental_libraries__"
```

Comment:
are you sure you want to drop: `experimental-opentelemetry` and `experimental-storage-grpc` ?

Reply:
Fixed.